web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -205,18 +205,12 @@ function AreaGraph({
   );
   const mouseOutHandler = useMemo(
     () => () => {
-      setGraphIndex(null);
-      setSelectedLayerIndex(null);
+      setGraphIndex(Number.NaN);
+      setSelectedLayerIndex(Number.NaN);
     },
     [setGraphIndex, setSelectedLayerIndex]
   );
 
-  // Don't render the graph at all if no layers are present
-  if (isEmpty(layers)) {
-    console.error('No layers present in AreaGraph');
-    return null;
-  }
-
```

Comment:
We should exit way before this is even run so this should never be a problem.

Reply:
We should exit and not render the graph at all if there is no data for it. This is handled when the graphs are created, not in this file itself. Or at least should be.